gems/pending/disk/modules/MSVSDiffDisk.rb

```diff
@@ -50,7 +50,6 @@ def d_init
       #
       # TODO: need details on Mac Alias Blob.
       #
-      # getParentPathMac(locator)
```

Comment:
I point it out because newcomers might actually look at this and think that the MacX case is actually used when it may not be and never will be.  We can always resurrect it if needed.